features/doctrine/search_filter.feature

```diff
@@ -1048,7 +1048,7 @@ Feature: Search filter on collections
   Scenario: Filters can use UUIDs
     Given there is a group object with uuid "61817181-0ecc-42fb-a6e7-d97f2ddcb344" and 2 users
     And there is a group object with uuid "32510d53-f737-4e70-8d9d-58e292c871f8" and 1 users
-    When I send a "GET" request to "/issue5735/issue5735_users?groups[]=/issue5735/groups/61817181-0ecc-42fb-a6e7-d97f2ddcb344&groups[]=/issue5735/groups/32510d53-f737-4e70-8d9d-58e292c871f8"
+    When I send a "GET" request to "/issue5735/issue5735_users?groups[]=61817181-0ecc-42fb-a6e7-d97f2ddcb344&groups[]=/issue5735/groups/32510d53-f737-4e70-8d9d-58e292c871f8"
```

Comment:
don't change a test please

Reply:
sorry, i was doing that at the same time as https://github.com/api-platform/core/pull/5760 , i'll make separate tests.